### PR TITLE
sendTx rpc

### PR DESCRIPTION
**Pull request by lzl124631x**
* Set `GenesisShardSize = 5`
* Run `./test/deploy.sh ./test/configs/ten-oneshard.txt`
* `http://localhost:9015` is the endpoint.

Search for `TODO(ricl): placeholder`. I left two stubs waiting for you to add.

---

Mirror PR of https://github.com/harmony-one/harmony/pull/850

### CODE REVIEW

File: core/tx_pool.go [hunk 1]
```diff
@@ -690,6 +691,13 @@ func (pool *TxPool) add(tx *types.Transaction, local bool) (bool, error) {
 	return replace, nil
 }
 
+// Add adds a transaction to the pool if valid and passes it to the tx relay
+// backend
+func (pool *TxPool) Add(ctx context.Context, tx *types.Transaction) error {
+	// TODO(ricl): placeholder
```
Comment (LeoHChen) [inline core/tx_pool.go]: @mikedoan , please chime in on how to implement this one.

File: internal/hmyapi/account.go [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+// passwords and are therefore considered private by default.
+type PrivateAccountAPI struct {
+	am        *accounts.Manager
+	nonceLock *AddrLocker
+	b         *core.HmyAPIBackend
+}
+
+// NewAccount will create a new account and returns the address for the new account.
+func (s *PrivateAccountAPI) NewAccount(password string) (common.Address, error) {
+	// TODO: port
```
Comment (LeoHChen) [inline internal/hmyapi/account.go]: I can handle this using keystore.

Comment (LeoHChen) [reply internal/hmyapi/account.go]: but who and how to keep the private key?

File: internal/hmyapi/harmony.go [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+
+// Syncing returns false in case the node is currently not syncing with the network. It can be up to date or has not
+// yet received the latest block headers from its pears. In case it is synchronizing:
+// - startingBlock: block number this node started to synchronise from
+// - currentBlock:  block number this node is currently importing
+// - highestBlock:  block number of the highest block header this node has received from peers
+// - pulledStates:  number of state entries processed until now
+// - knownStates:   number of known state entries that still need to be pulled
+func (s *PublicHarmonyAPI) Syncing() (interface{}, error) {
+	// TODO(ricl): port
```
Comment (LeoHChen) [inline internal/hmyapi/harmony.go]: @chaosma , please chime in on the implementation details.

File: node/node.go [hunk 4]
```diff
@@ -243,6 +246,15 @@ func New(host p2p.Host, consensusObj *consensus.Consensus, db ethdb.Database, is
 		// Consensus and associated channel to communicate blocks
 		node.Consensus = consensusObj
 
+		// TODO(ricl): placeholder. Set the account manager to node.accountManager
```
Comment (LeoHChen) [inline node/node.go]: I'll take a look. @LeoHChen

**Comment by lzl124631x** [comment]
> Please provide some more details or maybe create a readme file on how to test the RPC services.

@LeoHChen I'll update it in the next iteration since FS is also updating theirs, I need to sync with them about it.

**Pull request merged**